src/components/HomePage.jsx

```diff
@@ -223,256 +297,253 @@ class HomePage extends Component {
+										email our dev team
+									</Link>
+									.
+								</p>
+								<div className={classes.buttonDiv}>
+									<Button
+										href={config.incoreFAQUrl}
+										variant="contained"
+										color="primary"
+										className={classes.button}
+										startIcon={<QuestionAnswerIcon />}
+									>
+										Read FAQs
+									</Button>
+									<Button
+										href={config.slackInvitationLink}
 										target="_blank"
-										variant="contained" color="primary"
+										variant="contained"
+										color="primary"
 										className={classes.button}
-										style={{marginRight: "1em"}}
-								>
-									<img src="/public/slack-logo.png" style={{ height: "1em", marginRight: "8px"}}/>
-									Join Slack
-								</Button>
-								<Button href={`mailto:${config.mailingList}`}
+										style={{ marginRight: "1em" }}
+									>
+										<img
+											src="/public/slack-logo.png"
+											style={{ height: "1em", marginRight: "8px" }}
+										/>
+										Join Slack
+									</Button>
+									<Button
+										href={`mailto:${config.mailingList}`}
 										variant="contained"
 										color="primary"
 										className={classes.button}
 										startIcon={<MailOutlineIcon />}
-								>
-									Email Us
-								</Button>
-							</Box>
-						</Box>
-
-						{/*if version exists, display version; otherwise just the text*/}
-						{this.state.githubVersions && this.state.githubVersions["in-core"] ? (
-							<Chip
-								color="secondary"
-								size="medium"
-								className={classes.releaseChip}
-								label={`IN-CORE ${this.state.githubVersions["in-core"]} IS RELEASED INCLUDING`}
-							/>
-						) : (
-							<Chip
-								color="secondary"
-								size="medium"
-								className={classes.releaseChip}
-								label={"IN-CORE IS RELEASED INCLUDING"}
-							/>
-						)}
-
-						<div className={classes.versionSection}>
-							{this.state.repos.map((repo) => (
-								<div className={classes.versionLine}>
-									<Typography variant="body1" className={classes.versioning}>
-										{repo.title}
-
-										{/*if version exists, display version; otherwise not displaying the chip*/}
-										{this.state.githubVersions && this.state.githubVersions[repo.repoName] ? (
-											<Chip
-												size="small"
-												color="primary"
-												label={this.state.githubVersions[repo.repoName]}
-												className={classes.versioning}
-											/>
-										) : null}
-
-										{Object.keys(repo.options).map((option) => (
-											<Link
-												color="primary"
-												underline="always"
-												className={classes.versioning}
-												href={repo.options[option]}
-												target="_blank"
-											>
-												{option}
-											</Link>
-										))}
-									</Typography>
+									>
+										Email Us
+									</Button>
 								</div>
-							))}
+								<Divider />
```

Comment:
I would remove the last divider since it's the bottom of the screen. But up to you.